src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1552,21 +1557,32 @@ void TorrentHandle::handleTrackerErrorAlert(const libtorrent::tracker_error_aler
 void TorrentHandle::handleTorrentCheckedAlert(const libtorrent::torrent_checked_alert *p)
 {
     Q_UNUSED(p);
-    qDebug("%s have just finished checking", qUtf8Printable(hash()));
+    qDebug("\"%s\" have just finished checking", qUtf8Printable(name()));
 
-    updateStatus();
+    if (m_startupState == NotStarted) {
```

Comment:
I am looking this online: How can the program flow reach this point as true?

Reply:
When torrent is restored as "resumed" (i.e. it should be resumed after initialization) program flow reaches this point as true.